hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/io/compress/LzopCodec.java

```diff
@@ -0,0 +1,108 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.hadoop.io.compress;
+
+import io.airlift.compress.lzo.LzoCodec;
+import org.apache.hadoop.conf.Configurable;
+import org.apache.hadoop.conf.Configuration;
+
+import java.io.IOException;
+
+public class LzopCodec extends io.airlift.compress.lzo.LzopCodec
+        implements Configurable, CompressionCodec {
+    @Override
+    public Class<? extends Compressor> getCompressorType()
+    {
+        return LzopCodec.HadoopLzopCompressor.class;
```

Comment:
I see that `createCompressor` returns `HadoopLzoCompressor()`. Should we keep these two in sync?

Reply:
Good catch. Ideally yes, but `HadoopLzoCompressor` is private in airlift, so I can not easily do it. I added a new override
```
    @Override
    public Compressor createCompressor()
    {
        return new HadoopLzopCompressor();
    }
```
to keep them in sync, and I will try to work with aircompressor to move this to their codebase.